src/store/types/index.ts

```diff
@@ -4,12 +4,12 @@ import {
   TRemoveFromArrayAction,
   TUpdateInArrayAction,
   TPushIntoArrayAction,
-  IForm,
```

Comment:
Could we export IForms from here as well so that we know the index of `store/types` can always be referenced to get the appropriate type?